monitor/monitor.go

```diff
@@ -48,8 +48,8 @@ type MonitorOptions struct {
 
 // Valid enforces that a MonitorOptions instance is valid. There must be
 // a non-empty LogURI and LogKey. One of FetchOpts or SubmitOpts must not be
-// nil. If the FetchOpts are not nil, they must be valid. If the SubmitOpts are
-// not nil, they must be valid.
+// non-nil and valid. If the FetchOpts are not nil, they must be valid. If the
```

Comment:
My thought with the "non-nil and valid" change was that we could delete the sentences "If X is not nil, must be valid". Makes the overall comment shorter and easier to understand.

Reply:
Ahh, ok. I think 4feff79 is what you want then.